src/core/stompest/config/__init__.py

```diff
@@ -1,3 +1,63 @@
+from _ssl import CERT_REQUIRED, CERT_NONE
+from ssl import _DEFAULT_CIPHERS
+
+
+class SSLConfig(object):
```

Comment:
It doesn't feel right that SSL and non-SSL stuff is mixed in this module. But maybe I'm wrong.